tools/autograd/derivatives.yaml

```diff
@@ -272,12 +262,10 @@
   self: std::get<0>(gesv(grad, A.t()))
   A: -at::mm(std::get<0>(gesv(grad, A.t())), solution.t())
 
-# get_device
-
-- name: gt(Tensor self, Scalar other)
+- name: gt_(Tensor self, Scalar other)
   self: zeros_like(self)
 
-- name: gt(Tensor self, Tensor other)
+- name: gt_(Tensor self, Tensor other)
```

Comment:
Why do we need these specs? If an output never requires grad then the formulas don't matter anyway, right?

Reply:
Out-of-place `gt` automatically doesn't require grad because the return value is an IndexTensor. However, in-place `gt_` will return a `FloatTensor` if the input is a `FloatTensor`.